driver-core/src/main/com/mongodb/internal/async/client/AsyncMapReduceIterableImpl.java

```diff
@@ -148,12 +148,14 @@
     }
 
     @Override
+    @Deprecated
```

Comment:
No need to deprecate internal classes.  Just creates need for otherwise unnecessary warning suppressions

Reply:
Done